recipe/yum_requirements.txt

```diff
@@ -1,8 +1,5 @@
 csh
-imake
 libX11-devel
 libXaw-devel
 libXmu-devel
```

Comment:
These are still needed b/c the config script could not find the headers from the conda package.